api/src/pcapi/local_providers/titelive_things/titelive_things.py

```diff
@@ -239,6 +240,11 @@ def __next__(self) -> list[ProvidableInfo] | None:
             elements = next(self.data_lines).split("~")  # type: ignore [arg-type]
 
         if len(elements) != NUMBER_OF_ELEMENTS_PER_LINE:
+            logger.error(
+                "SyncError: number of elements mismatch. (expected: %s, actual: %s)",
+                len(elements),
+                NUMBER_OF_ELEMENTS_PER_LINE,
```

Comment:
(détail) il me semble qu'il y a une inversion.

```suggestion
                "SyncError: number of elements mismatch. (expected: %s, actual: %s)",
                NUMBER_OF_ELEMENTS_PER_LINE,
                len(elements),
```

Reply:
merci !